templates/person/show.json.twig

```diff
@@ -0,0 +1,94 @@
+{
+    "@context": "https://schema.org",
+    "@graph": [
+        {
+            "@type": "Person"
+            {% if person.firstname %},"givenName": "{{ person.firstName}}" {% endif %}
+            {% if person.lastName %},"familyName": "{{ person.lastName}}" {% endif %}
+            ,"gender": "{% include 'person/gender.html.twig' %}"
+            {% if (person.wikipediaUrl) or (person.viafUrl) %}
+            ,"sameAs": [
+                {% if person.wikipediaUrl %} "{{ person.wikipediaUrl}}"{% endif %}
+                {% if (person.wikipediaUrl) and (person.viafUrl) %} , {% endif %}
+                {% if person.viafUrl %} "{{ person.viafUrl}}" {% endif %}
+            ]
+            {% endif %}
+            {% if person.dob %},"birthDate": "{{person.dob}}" {% endif %}
+            {% if person.dod %}, "deathDate": "{{person.dod}}" {% endif %}
+            {% if person.cityOfBirth %},
+                "birthPlace": {
+                    "@type": "Place",
+                    "name": "{{person.cityOfBirth}}",
+                    "geo": {
+                        "@type": "GeoCoordinates",
+                        "latitude": "{{person.cityOfBirth.latitude}}",
+                        "longitude": "{{person.cityOfBirth.longitude}}"
+                    }
+                }
+             {% endif %}
+            {% if person.cityOfDeath %},
+                "deathPlace": {
+                    "@type": "Place",
+                    "name": "{{person.cityOfDeath}}",
+                    "geo": {
+                        "@type": "GeoCoordinates",
+                        "latitude": "{{person.cityOfDeath.latitude}}",
+                        "longitude": "{{person.cityOfDeath.longitude}}"
+                    }
+                }
+            {% endif %}
+            ,"@id": "{{ url('person_show', {'id':person.id}) }}"
+        }
+        {% for titleRole in person.getTitleRoles("Author") %}
```

Comment:
person.getTitleRoles() expects true or false, not a string. If you pass true, the title roles will be sorted. Passing a string here will result in an error in a future version of php which will be more strict.

Reply:
@ubermichael , so can you please help me with how to filter on those title roles who I want the person to be only the author of? 
I tried different things and each usually ended in errors except this (which is partly due to lack of my knowledge about title roles and the relations)?